es-ds-components/components/es-modal.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import PrimeDialog from "primevue/dialog";
+
+const props = defineProps({
+    hideFooter: {
+        type: Boolean,
+        default: false,
+    },
+    size: {
+        type: String,
+        default: "lg",
```

Comment:
Can we add the validator from the old one?
```
validator: (val) => ['sm', 'md', 'lg', 'xl'].includes(val),
```
I also notice that the default size has changed from `md` to `lg`- is this something we want?